### ISSUE TITLE
New UI is slower

### ISSUE THREAD

**Issue by grdoor** [microsoft/vscode#60419]
Issue Type: <b>Bug</b>

The new chrome is pretty, but I've noticed startup time when opening files seems slower and performance of visuals when resizing lags in a way I never noticed before. Please improve performance of the new UI.

VS Code version: Code 1.28.0 (431ef9da3cf88a7e164f9d33bf62695e07c6c2a9, 2018-10-05T14:58:53.203Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.79GB (11.05GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
githistory|don|0.4.3
xml|Dot|2.3.2
EditorConfig|Edi|0.12.4
auto-close-tag|for|0.5.6
mssql|ms-|1.4.0
azure-account|ms-|0.4.3
cpptools|ms-|0.19.0
csharp|ms-|1.16.2
PowerShell|ms-|1.9.0
qub-msbuild|qub|0.6.3
vscode-icons|rob|7.27.0
vscode-hexdump|sle|1.7.0
usql-vscode-ext|usq|0.2.14

</details>

**Comment by Owchzzz** [comment]
Yeah noticed this as well. try switching out to the default Theme. there might be changes in the theme rendering. I switched back to the default light theme which seemed to have fixed the lag for now. 

OS: Mac Os High Sierra

**Comment by Owchzzz** [comment]
@sbatten stable. im running a macbook air 2017

**Comment by lucasmarotta** [comment]
To me since version 1.27, to open the app takes **44s seconds to launch**, with fresh install without any extension installed. I tried to swich the title bar to native, but I also obtained the same result. To open files, even the blank ones, can take up to 1.3 minutes. I am using fresh Windows 10 installation with a i3 4130T. The problem is that using the version 1.25 it's **much faster** (launch in 2-3s).

Tested with both 1.27 and 1.28 64 bits.

**Comment by sbatten** [comment]
@grdoor @Owchzzz @lucasmarotta going through my issues and noticed this one has slipped on my end. Perf issues relating to startup have been addressed over the life of this issue and I would like to get your thoughts on the current state. If you are still seeing very slow startup, first try as @lucasmarotta suggests, switching to native to compare. If you have the same result, I'll adjust accordingly.

**Comment by grdoor** [comment]
Start up time when opening files seems improved to me, I haven't noticed it as too slow for some time (though, admittedly, it's been long enough that I may have acclimated.) 

The resizing issue is still there, though. I've been using the default (dark) theme all along.  I also just tested and screencapped using the default light theme. I notice latency when resizing the outer window frame between the window contents and the frame itself, though it does catch up within a fraction of a second. This may be related to my use of RDP for most development work, but I did start noticing it much more after this update (perhaps it was always there, but now there're a few more milliseconds of latency that are getting magnified or something like that?)

I've attached two images of what I see (contents themselves are blurred)

![lagwhendraggingdownandright](https://user-images.githubusercontent.com/7320226/52145389-ad967c00-2615-11e9-97e5-9db4a210271a.png)
![lagwhendraggingupandleft](https://user-images.githubusercontent.com/7320226/52145396-aff8d600-2615-11e9-87fc-20b8b11e8c2c.png)

When enlarging a window by dragging the bottom right corner down and right, I see black bars where the window frame is but the inner content hasn't expanded to, yet.

Likewise, when shrinking a window by dragging the bottom right corner up and to the left, I see the contents of the window get cropped by the window frame until they catch up.
